mozdownload/scraper.py

```diff
@@ -242,6 +243,51 @@ def detect_platform(self):
         else:
             return "%s%d" % (mozinfo.os, mozinfo.bits)
 
+    def get_checksum_url(self):
+        """Get the checksum url for current version."""
+        filename = 'SHA512SUMS'
+        url = os.path.split(os.path.split(os.path.split(self.url)[0])[0])[0] + '/' + filename
+        return url
```

Comment:
It is for the candidate build. archive.mozilla.org/pub/firefox/candidates/53.0b4-candidates/build1/ contains the 'SHA512SUMS' file which contains all the checksums